src/core/config/OperationConfig.js

```diff
@@ -4018,6 +4019,29 @@ const OperationConfig = {
         inputType: "string",
         outputType: "number",
         args: []
+    },
+    "Set Operations": {
+        module: "Default",
+        description: "Performs set operations",
+        inputType: "string",
+        outputType: "html",
```

Comment:
Not sure why this is `html`? It looks like your output is always just a simple string.